src/lib.rs

```diff
@@ -520,23 +517,23 @@ impl BasicDataFormatDescriptorHeader {
     }
 }
 
-pub struct BasicDataFormatDescriptor<'data> {
-    pub header: BasicDataFormatDescriptorHeader,
+pub struct DfdDescriptorBasic<'data> {
```

Comment:
This stutters. Maybe `DfdBlockBasic`?

Reply:
Oh I thought it was an actual descriptor of the descriptor, but it's not.

Reply:
Updated.